tests/unit/components/buttons/HdRadioButton.spec.js

```diff
@@ -11,7 +11,9 @@ describe('HdRadioButton', () => {
   const name = 'lorem';
   const value = 'ipsum';
   const label = 'dolor';
-  const mockedSelect = jest.fn();
+  const ICON_SELECTOR = '.radioButton__icon';
+  const BACKGROUND_STYLE = 'background-image: url(desktop-icon-url);';
+  const EVENT_EMITTED = 'select';
```

Comment:
This is unnecessary IMHO but not a big deal.

Reply:
I've moved the EVENT_EMITTED to the scope of the test using it, but kept the variable since I'm using the same string twice.